docs/support-bundle-guide.md

```diff
@@ -0,0 +1,183 @@
+in section [Use case example](#use-case-example).
+
+## Table of Contents
+
+<!-- toc -->
+- [Prerequisites](#prerequisites)
+- [Use case example](#use-case-example)
+  - [Running antctl commands](#running-antctl-commands)
+  - [Applying SupportBundleCollection CRD](#applying-supportbundlecollection-crd)
+- [List of collected items](#list-of-collected-items)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## Prerequisites
+
+SupportBundle Collection is introduced in Antrea v1.10.0 as an alpha feature. The feature gate
+`SupportBundleCollection` must be enabled in the `antrea-agent` configuration to use the feature.
+If you are planning to collect support bundle on the specified ExternalNode, you should enable
+`SupportBundleCollection` on that ExternalNode as well.
+
+```yaml
+  antrea-agent.conf: |
+    featureGates:
+    # Enable collecting support bundle files with SupportBundleCollection CRD.
+    SupportBundleCollection: true
+```
+
+## Use case example
+
+### Running antctl commands
+
+Please refer to the corresponding [antctl user guide section](antctl.md#collecting-support-information).
+
+### Applying SupportBundleCollection CRD
+
+In this section, we will create two SupportBundleCollections for Antrea Nodes and External Node.
+Assume we have a cluster with Nodes named "kube-worker" and "kube-master". In addition, we have
+set up an ExternalNode named `vm` in `vm-ns` namespace by following the instruction
+in [Install ExternalNode on VM](external-node.md#install-antrea-agent-on-vm).
+
+A Secret need to be created first with the username and password of the SSH Server.
+The Secret will be used in `authSecret` in the following YAML examples.
+
+```bash
+kubectl create secret generic support-bundle-secret  --from-literal=username='yoursshusername'  --from-literal=password='yoursshpassword'
+```
+
+Then we could apply the following YAML files.
+The first one is to collect support bundle on Antrea Nodes "kube-worker" and "kube-master".
+The second one is to collect support bundle on ExternalNode "vm".
+
+```yaml
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-kube-nodes
+spec:
+  nodes:
+    nodeNames: // empty list means all the Nodes
+      - kube-master
+      - kube-worker
+  expirationMinutes: 10 // expirationMinutes is the requested duration of validity of the SupportBundleCollection. A SupportBundleCollection will be marked as Failed if it does not finish before expiration.
+  fileServer:
+    url: sftp://yourtestdomain.com:22/root/test 
+  authentication:
+    authType: "BasicAuthentication"
+    authSecret:
+      name: support-bundle-secret
+      namespace: default
+```
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: SupportBundleCollection
+metadata:
+  name: collect-support-bundle-for-vm-external-node
+spec:
+  externalNodes:
+    nodeNames: // empty list means all the ExternalNodes in namespace
+      - vm
```

Comment:
I think you should also give an introduction that we support using `                    nodeSelector` to select Nodes/ExternalNodes by labels?

Reply:
Also need your help for this comment, could you provide a proper description and sample for this? thanks.

Reply:
We can add this sample in "externalNodes" block,

```
    nodeNames: # Empty list means all the ExternalNodes in the Namespace
      - vm 
    nodeSelector:
       matchLabels:
         app: db   # Collect support bundle files from the ExternalNodes which are configured with label "app: db" in the Namespace
    namespace: vm-ns
```

Reply:
updated. Thanks